lapce-proxy/src/lib.rs

```diff
@@ -20,11 +20,17 @@ use lapce_rpc::{
 };
 use once_cell::sync::Lazy;
 
-#[cfg(debug_assertions)]
-pub const APPLICATION_NAME: &str = "Lapce-debug";
+pub static APPLICATION_NAME: Lazy<&str> = Lazy::new(application_name);
 
-#[cfg(not(debug_assertions))]
-pub const APPLICATION_NAME: &str = "Lapce";
+fn application_name() -> &'static str {
+    if cfg!(debug_assertions) {
+        "Lapce-Debug"
+    } else if option_env!("RELEASE_TAG_NAME") == Some("nightly") {
```

Comment:
They are nightly-shortsha

Reply:
Right, I forgot to check updated code in `version()`
Would below be ok for both `application_name()` and `version()`? 
```suggestion
    } else if option_env!("RELEASE_TAG_NAME").unwrap_or("").starts_with("nightly") {
```

IMO it's simpler that current code:
```rs
    } else if option_env!("RELEASE_TAG_NAME").is_some()
        && option_env!("RELEASE_TAG_NAME")
            .unwrap()
            .starts_with("nightly")
    {
```

Reply:
Yes it is better.